config/root-phase0/apiresourceschema-clusterworkspaces.tenancy.kcp.dev.yaml

```diff
@@ -216,7 +216,7 @@ spec:
                 description: ClusterWorkspaceInitializer is a unique string corresponding
                   to a cluster workspace initialization controller for the given type
                   of workspaces.
-                pattern: ^root(:[a-z0-9]([-a-z0-9]*[a-z0-9])?)*(:[a-z][a-z0-9]([-a-z0-9]*[a-z0-9])?)$
+                pattern: ^(root(:[a-z0-9]([-a-z0-9]*[a-z0-9])?)*(:[a-z][a-z0-9]([-a-z0-9]*[a-z0-9])?))|(system:.+)$
```

Comment:
we are really over-ambitious here in status to specify such a pattern.

Reply:
This code is not net new. This is just a change to add system:* as a valid option. The initializer names are auto-set by our controllers, IIRC. Let's discuss outside of this PR if we want to change anything about this?